src/lean_interact/server.py

```diff
@@ -121,7 +121,7 @@ def _execute_cmd_in_repl(self, json_query: str, verbose: bool, timeout: float |
         with self._lock:
             if verbose:
                 logger.info("Sending query: %s", json_query)
-            self._proc.stdin.write(json_query + "\n\n")
+            self._proc.stdin.write(json_query + "\n\n\n\n")
```

Comment:
without `"\n\n\n\n"` I was getting hangs with my existing installation of REPL 

or maybe it's because `"\r\n\r\n"` is needed, as was discussed in https://github.com/leanprover-community/repl/pull/5#discussion_r1595010942 ?

Reply:
Thanks for the PR. I am on vacation now and won't have time to have a proper look into this before a week.

It seems that the REPL has changed since the PR you mention and the version you are using, hence `"\n\n"` is now enough. It is possible that `"\n\n\n\n"` or `"\r\n\r\n"` will not work with current REPL versions.

> @augustepoiroux I also noticed that you are using universal_newlines=True which seems the deprecated version of text=True, according to https://docs.python.org/3/library/subprocess.html

If I am not mistaken, we are using `text=True` in Leaninteract and not `universal_newlines=True`.

Reply:
(You are right, sorry about `universal_newlines` - I got confused with something else)